### PR TITLE
fix technique's damage modify no effect

### PR DESCRIPTION
**Pull request by 50301a**
bash,cut,stab isn't reference before

### CODE REVIEW

File: src/melee.cpp [hunk 1]
```diff
@@ -1169,10 +1169,13 @@ damage_unit &get_damage_unit( std::vector<damage_unit> &di, const damage_type dt
...
 {
-    auto bash = get_damage_unit( di.damage_units, DT_BASH );
-    auto cut  = get_damage_unit( di.damage_units, DT_CUT );
-    auto stab = get_damage_unit( di.damage_units, DT_STAB );
+    auto &bash = get_damage_unit( di.damage_units, DT_BASH );
+    auto &cut  = get_damage_unit( di.damage_units, DT_CUT );
+    auto &stab = get_damage_unit( di.damage_units, DT_STAB );
 
+    //std::string damage_str;
+    //damage_str = string_format(_("damage:%dbash,%dcut,%dstab,%dtotle"), (int)di.type_damage(DT_BASH), (int)di.type_damage(DT_CUT), (int)di.type_damage(DT_STAB), (int)di.total_damage());
```
Comment (Rivet-the-Zombie) [inline src/melee.cpp]: What is '%dtotle' about?

Comment (kevingranade) [inline src/melee.cpp]: I assume it's supposed to be %dtotal

Comment (Rivet-the-Zombie) [inline src/melee.cpp]: Oh, derp. I should've tried reading it out loud.

Comment (50301a) [inline src/melee.cpp]: ok

**Comment by BevapDin** [comment]
Why are the messages commented out? They are completely useless that way. You should either remove the commented out code completely, add enable it as a debug messages: `add_msg( m_debug, "..." )` - (`m_debug` makes it only appear when the debug mode is enabled).

**Comment by 50301a** [comment]
fix msg again

**Pull request merged**